config/_shared.json

```diff
@@ -140,5 +140,6 @@
 		{ "value": 448, "langSlug": "za", "name": "za - (Cuengh)", "wpLocale": "" },
 		{ "value": 449, "langSlug": "zh-cn", "name": "zh-cn - 简体中文", "wpLocale": "zh_CN", "popular": 13 },
 		{ "value": 452, "langSlug": "zh-tw", "name": "zh-tw - 繁體中文", "wpLocale": "zh_TW", "popular": 14 }
-	]
+	],
+	"project": "calypso"
```

Comment:
Renamed to `wordpress-com`, because of conflicting name with [wpcom.js](https://github.com/Automattic/wpcom.js).